nsapass

```diff
@@ -456,32 +434,31 @@ def chpass():
 # arguments are specified by their named arguments to `add_argument`.
 # 'PASSWORD' stands for `[-g LEN SET | -m]` and is handled separately.
 allcmds = [
-    ['get', ['username','index','-a','-z'],
+    ['get', ['id','rev','-o','-z'],
      {'help':'load a password to clipboard'}],
-    ['add', ['-a','-z','PASSWORD','-u','-i','-n'],
+    ['add', ['id','-z','PASSWORD','-n'],
```

Comment:
main reason i'm merging this 😃 - i like this idea of yours (it would've been sad if i had applied this change on my own without having it attributed to you).